dv/verilator/memutil/cpp/verilator_memutil.cc

```diff
@@ -228,9 +228,13 @@ bool VerilatorMemUtil::IsFileReadable(std::string filepath) const {
 bool VerilatorMemUtil::ElfFileToBinary(const std::string &filepath,
                                        uint8_t **data,
                                        size_t &len_bytes) const {
-  bool retval;
-  std::list<BufferDesc> buffers;
```

Comment:
Can you remove the BufferDesc struct definition in the beginning of this file, it's now unused.

Reply:
Good eyes! Fixed -- and thank you!